### PR TITLE
xero chart of accounts commit

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/xero/settings/xero-configurations/configuration/configuration.component.html [hunk 1]
```diff
@@ -89,13 +89,33 @@
...
         </div>
       </mat-slide-toggle>
-
+      
+      <div class="configurations--label" *ngIf="isChartOfAccountsEnabled">
+        Select Chart of Accounts to Import
+        <mat-icon
+          [inline]="true"
+          class="info-icon"
+          matTooltip="The selected accounts from Quickbooks Online will be imported as categories in Fyle in an enabled state. These categories can be disabled if needed from Fyle Admin settings"
```
Comment (Sravanksk) [inline src/app/xero/settings/xero-configurations/configuration/configuration.component.html:104]: rm `Quickbooks Online`

File: src/app/xero/settings/xero-configurations/configuration/configuration.component.ts [hunk 2]
```diff
@@ -121,7 +132,7 @@ export class ConfigurationComponent implements OnInit {
     const autoMapEmployees = that.generalSettingsForm.value.autoMapEmployees ? that.generalSettingsForm.value.autoMapEmployees : null;
     const autoCreateDestinationEntity = that.generalSettingsForm.value.autoCreateDestinationEntity;
     const importTaxCodes = that.generalSettingsForm.value.importTaxCodes ? that.generalSettingsForm.value.importTaxCodes : false;
-
+    const chartOfAccounts = importCategories ? this.generalSettingsForm.value.chartOfAccounts : ['Expense'];
```
Comment (Sravanksk) [inline src/app/xero/settings/xero-configurations/configuration/configuration.component.ts:135]: data you are sending to API for charts_of_accounts should be in capitalized to match the accounts API response 
eg: `['EXPENSE', 'REVENUE']`

File: src/app/xero/settings/xero-configurations/configuration/configuration.component.ts [hunk 3]
```diff
@@ -185,6 +196,7 @@ export class ConfigurationComponent implements OnInit {
     that.workspaceId = that.route.snapshot.parent.parent.params.workspace_id;
     that.settingsService.getXeroCredentials(that.workspaceId).subscribe((xeroCredentials: XeroCredentials) => {
       that.xeroCompanyCountry = xeroCredentials.country;
+      that.allAccountTypes = ['Expense', 'Asset', 'Equity', 'Liability', 'Revenue'];
```
Comment (Sravanksk) [inline src/app/xero/settings/xero-configurations/configuration/configuration.component.ts:199]: The list of account types should be CAPITALIZED I guess, pls check and update

Comment (DhaaraniCIT) [reply src/app/xero/settings/xero-configurations/configuration/configuration.component.ts:199]: it should be in camelcase, in qbo it is like that

File: src/app/xero/settings/xero-configurations/configuration/configuration.component.ts [hunk 4]
```diff
@@ -55,6 +55,9 @@ export class ConfigurationComponent implements OnInit {
         paymentsSyncOption = 'sync_xero_to_fyle_payments';
       }
 
+      const COA = that.generalSettings.charts_of_accounts.map(element => {
```
Comment (Sravanksk) [inline src/app/xero/settings/xero-configurations/configuration/configuration.component.ts:58]: can you add a better name for this instead of `COA`

**Review by Sravanksk — APPROVED** [review]

**Pull request merged**